src/react-elastic-carousel/components/Arrow.js

```diff
@@ -28,6 +28,7 @@ const Arrow = ({ direction, onClick, icons, style, ...rest }) => {
   };
   return (
     <Button
+      type="button"
```

Comment:
I would add that attribute in `Button` itself in case we will use it in other places:

```jsx
export default styled.button.attrs(({ type = "button" }) => ({ type }))`
  transition: all 0.3s ease;
  font-size: 1.6em;
  background-color: rgba(103, 58, 183, 0.1);
  // ...
`
```